src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Bindings/WatchableCloudBlobStream.cs

```diff
@@ -153,5 +187,40 @@ public ParameterLog GetStatus()
                 return null;
             }
         }
+
+        private void Log()
+        {
+            if (_logged)
+            {
+                return;
+            }
+
+            if (_blob == null || _logger == null)
+            {
+                return;
+            }
+
+            try
+            {
+                List<string> statisticsList = new List<string>
+                {
+                    $"BlobName: {_blob.Name}",
+                    $"ContainerName: {_blob.Container.Name}",
+                    $"ETag: {_blob.Properties.ETag}",
+                    $"BlobType: {_blob.BlobType}",
+                    $"BytesWritten: {_countWritten}",
+                    $"ElapsedTime: {_timeWrite.Elapsed}",
+                };
+
+                string statistics = string.Join(", ", statisticsList);
+                string logMessage = $"WriteStream ({statistics})";
+                _logger.LogInformation(logMessage);
+                _logged = true;
+            }
+            catch
```

Comment:
Not sure what exceptions this code might throw, but if its throwing I want to know about it so we can fix it. So if we are going to swallow the exception, at least _logger.LogError it.